examples/compute/sharedvpchostproject.yaml

```diff
@@ -3,11 +3,39 @@ kind: SharedVPCHostProject
 metadata:
   annotations:
     meta.upbound.io/example-id: compute/v1beta1/sharedvpchostproject
-    upjet.upbound.io/manual-intervention: "Required organizational level permissions"
```

Comment:
Hi @RemiVR,
Looks like these are resources that need organizational level permissions and access to multiple projects, which our testing infrastructure cannot properly handle as of now. I would suggest to add the `manual-intervention` annotation back (and also add the same annotation to the dependencies below) so that we disable these tests until we have the proper infrastructure in place to support these tests.

However, because you've successfully validated the configuration manually, I believe we can still merge this PR.

Reply:
Added the labels to example files.